### PR TITLE
ES: delete metric and delete directory

### PR DESCRIPTION
**Pull request by adericbourg**

### CODE REVIEW

File: biggraphite/drivers/elasticsearch.py [hunk 1]
```diff
@@ -486,14 +486,29 @@ def update_metric(self, name, updated_metadata):
         self.create_metric(updated_metric)
 
     def delete_metric(self, name):
-        # TODO: Implement
-        log.warn("update_metric is not implemented")
-        pass
+        name = bg_accessor.sanitize_metric_name(name)
+
+        query = elasticsearch_dsl.Search() \
```
Comment (iksaif) [inline biggraphite/drivers/elasticsearch.py]:
```
+        query = elasticsearch_dsl.Search() \
+            .using(self.client) \
+            .index("%s*" % self._index_prefix)
```

this seems duplicated, maybe we can add a helper to do that ?

Comment (adericbourg) [reply biggraphite/drivers/elasticsearch.py]: Sure. This is used in other methods: do you agree if I do that in another PR?

Comment (iksaif) [reply biggraphite/drivers/elasticsearch.py]: ok !

Comment (adericbourg) [reply biggraphite/drivers/elasticsearch.py]: Done with #383

**Pull request merged**